crates/symbolicator/src/cache.rs

```diff
@@ -220,7 +237,19 @@ impl Cache {
             false
         };
 
-        let is_negative = metadata.len() == 0;
+        let is_download_error = if DOWNLOAD_ERROR_MARKER.len() as u64 == metadata.len() {
```

Comment:
Extremely nitpicky, but I think this condition should be the other way around; `metadata.len()` is being compared to a fixed quantity.

Reply:
also flipped a very similar check for malformed to keep things consistent across the entire method